dbflute-runtime/src/main/java/org/dbflute/cbean/coption/LikeSearchOption.java

```diff
@@ -324,6 +330,15 @@ public boolean hasCompoundColumn() {
         return _compoundColumnList;
     }
 
+    public void setCompoundColumnNullAsEmpty() {
```

Comment:
The "set" is special word in Java. (used as getter/setter)
LikeSearchOption does not use "set" for option methods.
It uses e.g. optimize, split, escape, ... instead.

So I want to use "set" at least in this class.
How about treatCompoundColumnNullAsEmpty()?

While, treatsCompoundColumnNullAsEmpty() (for determination) already exists.
treatCompoundColumnNullAsEmpty(), treatsCompoundColumnNullAsEmpty() are very confused.
Internal determination methods in LikeSearchOption are almost isXxx(), e.g. isLikePrefix() isAsOrSplit().
So how about isTreatCompoundColumnNullAsEmpty();

Of course, also isTreat...() is not good name.
But an user method should be more simple than an internal method.

Reply:
> The "set" is special word in Java. (used as getter/setter)
> LikeSearchOption does not use "set" for option methods.
> It uses e.g. optimize, split, escape, ... instead.
> 
> So I don't want to use "set" at least in this class.
> How about treatCompoundColumnNullAsEmpty()?
> 
> While, treatsCompoundColumnNullAsEmpty() (for determination) already exists.
> treatCompoundColumnNullAsEmpty(), treatsCompoundColumnNullAsEmpty() are very confused.
> Internal determination methods in LikeSearchOption are almost isXxx(), e.g. isLikePrefix() isAsOrSplit().
> So how about isTreatCompoundColumnNullAsEmpty();
> 
> Of course, also isTreat...() is not good name.
> But an user method should be more simple than an internal method.

Noted with thanks. I understand.
How about "compoundsNullAsEmpty()" as the user method and "isNullCompoundedAsEmpty()" as the internal method ?
Then, I think the user method is more simple and the internal method matches "isXxx()" expression.

Reply:
and more natural english maybe
Null is compounded as empty? -> maybe good
treats compound column null as empty -> wrong (what is the object?)

Reply:
> How about "compoundsNullAsEmpty()" as the user method and 
> "isNullCompoundedAsEmpty()" as the internal method ?
Oh, OK, good. That's new for me.

Reply:
> How about "compoundsNullAsEmpty()" as the user method and
> "isNullCompoundedAsEmpty()" as the internal method ?
> Oh, OK, good. That's new for me.

renamed them.
https://github.com/dbflute/dbflute-core/pull/54/commits/3178d878b1e2f463866a5dabcca86c2172091b4e